Train-In-Blocks-Ltd/Landing
# feat: Exclusive blog content

# TIB Landing PR

## Changes

A brief list of changes

- Adjusted blog pages to require them to sign up before reading the post
- These blog posts can be made exclusive optionally on CMS
- Stores whether user is signed up or not in ```localStorage```
- If the user exists already, then show then content

## Test Cases

What should the reviewer do and expect to happen when testing

1. Head into to Worl Water Day blog post. **Expect to not see the content if you haven't submitted the email intake yet**
2. Use **test@mail.com** to unlock the content. **Expect to see the content**
3. Remove the local storage item ```existing-lead``` and refresh the page
4. Now sign up for a new email that's not in Mailchimp. **Expect to see the content**

## Demo

A screenshot or recording of the change.

<img width="1453" alt="Screenshot 2022-03-27 at 17 06 22" src="https://user-images.githubusercontent.com/63963445/160291136-0d989b69-b620-4e74-bb27-747b188789f8.png">
